### PR TITLE
set https upgrade to new endpoint and format

### PR DESCRIPTION
**Pull request by brindy**
Reviewer: Caine
Asana: 
CC:

**Description**:

New https endpoint.

New format for https (an array).

**Steps to test this PR**:

1. Run the tests
1. Launch the app and ensure domains are upgraded (e.g. english.sogou.com does not automatically redirect anyway so is a good test)
1. Browse around and confirm UX/performance is acceptable

### CODE REVIEW

**Comment by brindy** [comment]
@nilnilnil note that https://duckduckgo.com/contentblocking.js?l=https2 returns a 500 internal server error at the moment

**Review by nilnilnil — APPROVED** [review]
Lol.

**Comment by brindy** [comment]
Just saw the other PR for the server.  Will wait for that to get merged before merging this so I can give it one last chance locally.

**Pull request merged**
